Gemfile

```diff
@@ -86,6 +85,10 @@ group :development, :test do
   gem 'webmock'
 end
 
+group :lint, :development, :test do
+  gem 'rubocop-rails'
+end
```

Comment:
I have created a different group so that we can add other lint gems like `rubocop-performance`, `breakman` etc in the same group without needing to change the action.

I am open to change this to directly install the mentioned gems in action.